### PR TITLE
session: format some sql statements

### PR DESCRIPTION
**Pull request by bestgopher**
### What problem does this PR solve?

format some sql statements, sql keyword changed to uppercase。

### Release notes

- No Release Note

### CODE REVIEW

File: session/bootstrap.go [hunk 1]
```diff
@@ -122,166 +122,166 @@ const (
...
-      	bind_sql text NOT NULL ,
-      	default_db text  NOT NULL,
-		status text NOT NULL,
-		create_time timestamp(3) NOT NULL,
-		update_time timestamp(3) NOT NULL,
-		charset text NOT NULL,
-		collation text NOT NULL,
+		original_sql 	TEXT NOT NULL  ,
+      	bind_sql 		TEXT NOT NULL ,
+      	default_db 		TEXT NOT NULL,
```
Comment (zz-jason) [inline session/bootstrap.go:236]:
```suggestion
		bind_sql 		TEXT NOT NULL ,
		default_db 		TEXT NOT NULL,
```

**Comment by bestgopher** [comment]
> Good job! How did you find these SQL statements?

@zz-jason By reading the source code of tidb.

File: session/bootstrap.go [hunk 2]
```diff
@@ -122,166 +122,166 @@ const (
...
 	// See: https://dev.mysql.com/doc/refman/5.5/en/system-database.html#system-database-help-tables
-	CreateHelpTopic = `CREATE TABLE if not exists mysql.help_topic (
-  		help_topic_id int(10) unsigned NOT NULL,
-  		name char(64) NOT NULL,
-  		help_category_id smallint(5) unsigned NOT NULL,
-  		description text NOT NULL,
-  		example text NOT NULL,
-  		url text NOT NULL,
+	CreateHelpTopic = `CREATE TABLE IF NOT EXISTS mysql.help_topic (
+  		help_topic_id 		INT(10) UNSIGNED NOT NULL,
```
Comment (zz-jason) [inline session/bootstrap.go:152]: please convert the spaces to tabs.

**Comment by bestgopher** [comment]
@zz-jason Done!

File: session/bootstrap.go [hunk 3]
```diff
@@ -44,23 +44,23 @@ import (
...
-		Grant_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Grant_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
 		References_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Alter_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Alter_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Show_db_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Super_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Create_tmp_table_priv		ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Super_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Create_tmp_table_priv	ENUM('N','Y') NOT NULL DEFAULT 'N',
```
Comment (SunRunAway) [inline session/bootstrap.go:63]: Please make these align, also the other SQLs in this file

Comment (SunRunAway) [reply session/bootstrap.go:63]: Sorry, it's already done.

Comment (bestgopher) [reply session/bootstrap.go:63]: > Sorry, it's already done.
merge failed.

**Comment by zz-jason** [comment]
@bestgopher Thanks for your contribution, this PR is conflicted with the latest master branch, could you resolve the conflict and then we can merge it.

**Pull request closed without merging**